### PR TITLE
Add grip python package

### PR DESCRIPTION
**Pull request by k0ral**
###### Motivation for this change

Add grip python package.
###### Things done
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -7817,6 +7838,26 @@ in modules // {
     };
   };
 
+  path-and-address = buildPythonPackage rec {
+    version = "2.0.1";
+    name = "path-and-address-${version}";
+
+    src = pkgs.fetchFromGitHub {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: PyPI is typically preferred
https://pypi.org/project/path-and-address/

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -6861,6 +6861,27 @@ in modules // {
     };
   };
 
+  grip = buildPythonPackage rec {
+    version = "4.3.2";
+    name = "grip-${version}";
+
+    src = pkgs.fetchFromGitHub {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: PyPi is typically preferred
https://pypi.org/project/grip/

Unless the tests aren't included in the archive but are in the repo.

**Review by FRidh — CHANGES_REQUESTED** [review]
Judging from the Travis log, tests aren't run (0 tests). Please fix the test runner. If there are no tests, disable the tests and leave a comment with the reason.

File: pkgs/top-level/python-packages.nix (hunk 2 above)
Comment (k0ral) [inline pkgs/top-level/python-packages.nix]: Done.

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (k0ral) [inline pkgs/top-level/python-packages.nix]: Done.

**Comment by Mic92** [comment]
LGTM. Thanks.

**Comment by FRidh** [comment]
@Mic92 this was not yet ready to be merged.

**Comment by Mic92** [comment]
@FRidh did I miss something? Your concerns were fixed and I tested both locally.

**Comment by FRidh** [comment]
@Mic92 see https://github.com/NixOS/nixpkgs/pull/19019#pullrequestreview-1879203
As a bit of background. With Python we prefer to run the tests. The build reported it didn't find any tests. That can be because there aren't any, or because it just couldn't locate them. In the latter case the checkPhase needs to be fixed. And in the former, we explicitly disable the tests.

**Comment by Mic92** [comment]
sorry. For some reasons I only recognized the comments, not the resume above. This other pull request also enable tests for both.

**Pull request merged**